wiiu/vino/vino.ts

```diff
@@ -302,26 +389,26 @@ export default interface VinoAPI {
 	lyt_draw: (unk1: string, unk2: number, unk3: number, unk4: number, unk5: number, unk6: number) => void;
 
 	/**
-	 * Unknown
+	 * Resets the fixed red layout box
 	 */
 	lyt_reset: () => void;
 
 	/**
-	 * Unknown
-	 * @param unk1 - Unknown
-	 * @param unk2 - Unknown
-	 * @param unk3 - Unknown
-	 * @param unk4 - Unknown
+  	 * Draws a fixed red layout box (Seen in Japanese TVii)
+	 * @param left - Left position in screen
+	 * @param top -  Top position in screen
```

Comment:
Are these supposed to be "on the screen", not "in screen"? "In screen" doesn't make much sense

Also the `top` line has 2 extra spaces before the description